pom.xml

```diff
@@ -68,14 +68,6 @@
 
   <reporting>
     <plugins>
-      <plugin>
```

Comment:
I would remove all the `<reporting>` part because we do not build the maven site so there is no need for that as maybe we may have soon a dependabot pr for upgrading gitlog-maven-plugin plugin...

Reply:
This is out of scope here.